src/style.css

```diff
@@ -0,0 +1,115 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  src: url(./fonts/Avenir-Heavy.ttf);
+  src: url(./fonts/Avenir.ttc);
+}
+
+body {
+  font-family: "Avenir", monospace;
+}
+
+.page {
+  position: relative;
+  margin: 0 auto;
+  border-radius: 0;
+  box-sizing: border-box;
+  width: 100%;
+}
+
+.bigForm {
+  height: 70px;
+  margin: 20px 0;
+}
+
+.bigField {
+  position: relative;
+  height: 100%;
+}
+
+.bigBar {
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-sizing: border-box;
+  width: 100%;
+  padding: 23px 0 23px 62px;
+  background: #fff;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.bigBar::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+  font-family: "Avenir", monospace;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.bigBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.bigBar:focus {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  text-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-weight: 900;
+  outline: none;
+}
+
+.smallField {
+  position: relative;
+  height: 42px;
+  background: #fff;
+}
+
+.smallBar {
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  width: 100%;
+  height: 100%;
+  padding: 8px 0 10px 33px;
+  font-size: 14px;
+  line-height: 19px;
+  font-weight: 300;
+}
+
+.smallBar::placeholder {
+  font-size: 14px;
+  line-height: 19px;
+  font-family: "Avenir", monospace;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.smallBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
```

Comment:
you have repeatable styles. Move it to common class `bar` for example. Please improve your code style according this comment not only in these line but on whole your solution